android/src/app/organicmaps/routing/RoutingBottomMenuController.java

```diff
@@ -281,10 +280,21 @@ private void showRoutingDetails()
       return;
     }
 
-    Spanned spanned = makeSpannedRoutingDetails(mContext, rinfo);
+    boolean isHelicopter = RoutingController.get().isHelicopterRouterType();
+    Spanned spanned = makeSpannedRoutingDetails(mContext, rinfo, !isHelicopter);
     TextView numbersTime = mNumbersFrame.findViewById(R.id.time);
     numbersTime.setText(spanned);
 
+    TextView segmentsDesc = mNumbersFrame.findViewById(R.id.segments_desc);
+    if (isHelicopter && Framework.nativeGetRoutePoints().length > 2) {
+      RouteMarkData[] points = Framework.nativeGetRoutePoints();
+      segmentsDesc.setText(makeSpannedRoutePointsInfo(mContext, points));
+      segmentsDesc.setVisibility(View.VISIBLE);
+    }
+    else {
+      segmentsDesc.setVisibility(View.GONE);
+    }
```

Comment:
nit: Better to keep our common braces style here and below. Don't use braces for one-line expressions.

Reply:
Fixed